.circleci/config.yml

```diff
@@ -17,21 +17,34 @@ jobs:
             python -m venv /tmp/venv/openfisca_core
             echo "source /tmp/venv/openfisca_core/bin/activate" >> $BASH_ENV
 
+      - run:
+          name: Install base utils
+          command: pip install --upgrade pip setuptools wheel
+
       - run:
           name: Install dependencies
           command: |
-            make install
+            python setup.py egg_info  # Create a requirements file.
+            pip install `grep -v "^\[" *.egg-info/requires.txt`  # Install openfisca's dependencies.
             # pip install --editable git+https://github.com/openfisca/country-template.git@BRANCH_NAME#egg=OpenFisca-Country-Template  # use a specific branch of OpenFisca-Country-Template
             # pip install --editable git+https://github.com/openfisca/extension-template.git@BRANCH_NAME#egg=OpenFisca-Extension-Template  # use a specific branch of OpenFisca-Extension-Template
 
       - save_cache:
-          key: v1-py3-{{ .Branch }}-{{ checksum "setup.py" }}
+          key: v2-py3-{{ .Branch }}-{{ checksum "setup.py" }}
           paths:
             - /tmp/venv/openfisca_core
 
       - run:
-          name: Run Core tests
-          command: env PYTEST_ADDOPTS="--exitfirst" make test
+          name: Build & install openfisca-core
+          command: |
+            python setup.py bdist_wheel
+            find dist -name "*.whl" -exec pip install --force-reinstall --no-dependencies {} \;  # https://github.com/openfisca/openfisca-core/pull/1015#discussion_r687051988
+
+      - run:
+          name: Run openfisca-core tests
+          command: |
+            make clean check-syntax-errors check-style check-types
+            PYTEST_ADDOPTS="$PYTEST_ADDOPTS --cov=openfisca_core --exitfirst" pytest --pyargs openfisca_core.tests openfisca_web_api.tests  # Run the tests on the built & installed version of openfisca.
```

Comment:
Why do we use `pytest` instead of calling the `make` target?
Since #1040, this should be a `make` target.

Reply:
Sure, this is outdated.